src/shared/views/Mapper/CardBody.js

```diff
@@ -360,10 +360,20 @@ class _CardBody extends React.PureComponent<Props, State> {
         // tile of the screen
         let result = 0;
         if (this.tasksPerScreen) {
-            result = this.tasksPerScreen[screenNumber].reduce(
-                (sum, task) => sum + task.referenceAnswer,
-                0,
-            );
+            console.log('tps', this.tasksPerScreen.length, screenNumber);
+            // $FlowFixMe
+            if (this.tasksPerScreen[screenNumber]) {
+                result = this.tasksPerScreen[screenNumber].reduce(
```

Comment:
I'm a bit confused here. I assume that `result` refers to the number of tasks expected for this screen, as it is defined by the sum of the reference answers.

Reply:
Yes, it's the total number of taps expected on the screen, if the user does a "perfect sequence". I had put that in the comment at the top of the function, let me know if I should clarify